tests/shipping/models/Delivery - No city.model.js

```diff
@@ -0,0 +1,112 @@
+import { visitAndClearCookies, setup } from '../../../utils'
+import {
+  getRandomEmail,
+  fillEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { SKUS } from '../../../utils/constants'
+import {
+  selectCountry,
+  goToPayment,
+  interceptAutoCompleteResponse,
+} from '../../../utils/shipping-actions'
+import { payWithBoleto, completePurchase } from '../../../utils/payment-actions'
+
+export default function test(account) {
+  describe(`Delivery - No city - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.GLOBAL_PRODUCT], salesChannel: 2, account })
+
+      fillEmail(email)
+      fillProfile()
+
+      selectCountry('ARG')
+
+      cy.get('#ship-addressQuery').type('Castro Barros 523, Córdoba, Argentina')
+
+      cy.get('.pac-item')
+        .first()
+        .trigger('mouseover')
+
+      interceptAutoCompleteResponse({
+        address_components: [
+          {
+            long_name: '523',
+            short_name: '523',
+            types: ['street_number'],
+          },
+          {
+            long_name: 'Avenida Castro Barros',
+            short_name: 'Av. Castro Barros',
+            types: ['route'],
+          },
+          {
+            long_name: 'X5000HTF',
+            short_name: 'X5000HTF',
+            types: ['locality', 'political'],
+          },
+          {
+            long_name: 'Capital',
```

Comment:
can we change this so it is _really_ clear that this city isn't mapped and should be handled specially? something like

```suggestion
            long_name: 'Cidade Inexistente (não mudar pls)',
```